tic_tac_toe/game.py

```diff
@@ -63,42 +78,106 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
+    
+    set_of_combos = ( #row wins
```

Comment:
This general method is valid, but given that the rules for winning in the game have a bit more structure, I'd have used that instead of brute forcing it here. (ie, use various loops to generate the possible wins, letting the code reflect the structure of the problem. Again, more a style thing, but I think in this case would help readability/reasoning-ability a lot)